governance-proposal/TSC-Charter-Draft.md

```diff
@@ -0,0 +1,103 @@
+## ***DRAFT for Public Comment***
+
+# Technical Steering Committee (TSC) Charter
+
+## Section 1. Guiding Principle.  
+
+The Node.js Foundation will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 2. Evolution of Node.js Foundation Governance.
+
+Most large, complex open source communities have both a business and a technical governance model. Node.js Foundation’s technical leadership contains both a Technical Steering Committee (“TSC”) and Maintainers for major components or subsystems. Node.js Foundation’s business leadership is instantiated in a Board of Directors (the “Board”).  
+
+This Technical Steering Committee Charter reflects a carefully constructed balanced role for the TSC and the Board in the governance of Node.js Foundation. The charter amendment process is for the TSC to propose changes using simple majority of the full TSC, the proposed changes being subject to review and approval by the Board. The Board may additionally make amendments to the TSC charter at any time, though the Board will not interfere with day-to-day discussions, votes or meetings of the TSC.
+
+## Section 3. Board’s Role in Setting Node.js Foundation’s Strategic Direction.  
+
+The Board will set the overall TSC Policy. The policy will describe the overarching scope of the Node.js Foundation initiative, Node.js Foundation’s technical vision and direction and project release expectations in the form of expected cadence and intent. The Board will use the TSC as a delegate body for governing technical implementation, individual project scope and direction while they remain within the scope and direction of the policies as described in the TSC Policy document and approved by the Board.
```

Comment:
The Board is the legal fiduciary for running the independent legal entity that will be the Foundation. It's a corporate governance matter. We try to setup the TSC to run as independently as possible, but there are some things we just cannot avoid based on corporation requirements and responsibilities of a Board.  "Non-profit organizations" are filed as (typically Delaware) corporations, then they seek IRS approval for 501(c)(3) or 501(c)(6) non-profit status. As a corporation, they need a Board that is responsible for the entity. By way of an extreme example, if the TSC decided to change the TSC Policy to focus the community on developing nuclear and biological weapons, the Board of the entity would ultimately be responsible for the outcome ;-)